full-stack-tests/backend/package.json

```diff
@@ -40,7 +40,7 @@
     "@bentley/cif-subsurface-schema": "^1.0.3",
     "@bentley/cif-subsurface-conflict-analysis-schema": "^1.0.3",
     "@bentley/cif-units-schema": "^1.0.1",
-    "@bentley/imodeljs-native": "3.5.6",
+    "@bentley/imodeljs-native": "3.5.8",
```

Comment:
never thought about it til now, but why does this full-stack-test have a direct-dep on imjs-native?
shoudlnt this be a transitive dep via core-backend?

Reply:
My initial guess is that we used to be importing something from imodeljs-native directly, something that wasn't re-exported by the core-backend.ts file. Maybe it was removed in the last few months and this is no longer necessary.

Reply:
could you remove this dep, and lets see if anything breaks? i doubt it will but would be goof to know

Reply:
I updated the wiki to no longer call out full-stack-tests and removed the line from the .sh script which automatically updates the package versions